openapi/connector_mgmt-private-admin.yaml

```diff
@@ -879,6 +879,86 @@ paths:
                 $ref: "#/components/schemas/ConnectorAvailableOperatorUpgrade"
         required: true
 
+  /api/connector_mgmt/v1/admin/kafka_connector_types:
+    get:
+      tags:
+        - Connector Types
+      security:
+        - Bearer: [ ]
+      operationId: getConnectorTypes
+      summary: Returns a list of connector types
+      description: Returns a list of connector types
+      parameters:
+        - $ref: "connector_mgmt.yaml#/components/parameters/page"
+        - $ref: "connector_mgmt.yaml#/components/parameters/size"
```

Comment:
Can you also add `search` and `orderBy` parameters to make it uniform with all the other list operations? Maybe we can then also use it in `rhoc cli` (if it doesn't do it already) to help filter and order results.

Reply:
fixed